client/src/assets/i18n/de.po

```diff
@@ -2948,12 +2939,21 @@ msgstr "Veranstaltungstitel"
 msgid "Meetings"
 msgstr "Veranstaltungen"
 
+msgid "Meetings effected:"
```

Comment:
~~effected~~
affected

noticed while scrolling through #3834 ... Maybe this will be remembered next time :P